HighwayPatrol.cpp

```diff
@@ -10,9 +15,8 @@ void HighwayPatrol::scanHighway(Highway* h)
 {
     std::cout << name << ": scanning highway for speeders" << std::endl;
 
-    for( int i = h->vehicles.size(); --i >= 0; )
+    for( auto* v : h->vehicles )
```

Comment:
I'm not sure if I made this loop change or if you did, but it should be this, because of the removal from the vector in in `pullOver()`
```
for( size_t i = h->vehicles.size(); --i != 0; )
{
    auto* v = h->vehicles[i];
```
- [ ] please update.

Reply:
I didn't make this change